src/main/java/org/opentree/utils/GeneralUtils.java

```diff
@@ -119,9 +120,6 @@ public static String newickName (String origName) {
 		Boolean needQuotes = false;
 		String newickName = origName;
 		
-		// replace all spaces with underscore
-		newickName = newickName.replaceAll(" ", "_");
-		
 		// replace ':' with '_'. a hack for working with older versions of dendroscope e.g. 2.7.4
 		newickName = newickName.replaceAll(":", "_");
```

Comment:
Can we get rid of this dendropscope hack now? Just commenting out the line would fix this newick escaping code.